test/elixir/test/security_validation_test.exs

```diff
@@ -12,19 +12,19 @@ defmodule SecurityValidationTest do
   @auth_headers %{
     jerry: [
       # jerry:mouse
-      authorization: "Basic amVycnk6bW91c2U="
+      authorization: "basic amVycnk6bW91c2U="
     ],
     tom: [
       # tom:cat
-      authorization: "Basic dG9tOmNhdA=="
+      authorization: "BAsIc dG9tOmNhdA=="
     ],
     spike_cat: [
       # spike:cat - which is wrong
       authorization: "Basic c3Bpa2U6Y2F0"
     ],
     spike: [
       # spike:dog
-      authorization: "Basic c3Bpa2U6ZG9n"
+      authorization: "BASIC c3Bpa2U6ZG9n"
```

Comment:
This seems like a [piggyback](https://stackoverflow.com/a/333883) to me. Can you please add a test case that would fail without your production code change, but passes with it? That way when we're changing code parts related to this fix, we have one specific test case that shows us what exactly is about to break and it is clearly listed in the test report.

Reply:
Interesting term haha. I will make this change.

Reply:
Added two new tests cases to test lowercase and mixed case "Basic". See https://github.com/apache/couchdb/pull/3637/commits/a6c2028adf8f717b3a9ec97835388a456fb1ce03.

Reply:
Perfect, thank you!